manifests/05-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: elasticsearch-operator
       containers:
         - name: elasticsearch-operator
-          image: quay.io/openshift/elasticsearch-operator:latest
```

Comment:
@smarterclayton from a release perspective, does it matter if this is 'quay', 'dockerhub', or nothing?  Here or in https://github.com/openshift/elasticsearch-operator/blob/master/manifests/image-references.  It is the name in the image-reference which is significant correct?